Tests/SwiftProtobufTests/Test_BinaryEncodingOptions.swift

```diff
@@ -0,0 +1,74 @@
+// Tests/SwiftProtobufTests/Test_BinaryEncodingOptions.swift - Tests for binary encoding options
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See LICENSE.txt for license information:
+// https://github.com/apple/swift-protobuf/blob/main/LICENSE.txt
+//
+// -----------------------------------------------------------------------------
+///
+/// Test for the use of BinaryEncodingOptions
+///
+// -----------------------------------------------------------------------------
+
+import Foundation
+import XCTest
+import SwiftProtobuf
+
+class Test_BinaryEncodingOptions: XCTestCase {
```

Comment:
```suggestion
final class Test_BinaryEncodingOptions: XCTestCase {
```

Reply:
Agreed, and updated. I had left it without `final` before because it looks like the rest of the test case classes are missing the keyword, so I figured that was intentional

Reply:
Just put up #1486 for this.